Framework/Parallel/src/IO/MultiProcessEventLoader.cpp

```diff
@@ -169,13 +180,7 @@ void MultiProcessEventLoader::load(
 
     // Assemble multiprocess data from shared memory
     assembleFromShared(eventLists);
-
-    // clean up shared memory
-    for (const auto &name : m_segmentNames)
-      ip::shared_memory_object::remove(name.c_str());
   } catch (std::exception const &ex) {
```

Comment:
Can remove the try/catch now?

Reply:
I am not so familiar with `std::rethrow_if_nested`. What is the advantage over not catching the exception here?